VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/Util/MaterialConversion.cpp

```diff
@@ -81,11 +81,17 @@ void CountOpacityMapPixels(const uint16* SrcColors, int32 SizeX, int32 SizeY, ui
 								 [](const uint16* Color) { return static_cast<float>(*Color) / 0xFFFF; });
 }
 
+void CountOpacityMapPixels(const uint32* SrcColors, int32 SizeX, int32 SizeY, uint32& BlackPixels, uint32& WhitePixels)

```

Comment:
I think we could directly use a double here (CountOpacityMapPixels is templated).

Reply:
Even though the function is templated, we still need to define the accessor. Therefore I will keep the helper function for defining the accessor.